packages/stargate/CUSTOM_PROTOBUF_CODECS.md

```diff
@@ -2,29 +2,33 @@
 
 As of [v0.40](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.0), the
 Cosmos SDK uses
-[protocol buffers](https://developers.google.com/protocol-buffers) as its
-standard serialization format for blockchain state and wire communication.
-CosmJS by default supports protocol buffer serialization for many of the
-standard queries and messages defined by the Cosmos SDK, as well as
+[protocol buffers](https://developers.google.com/protocol-buffers) (also known
+as "protobufs") as its standard serialization format for blockchain state and
```

Comment:
isn't it

```suggestion
as "protobuf") as its standard serialization format for blockchain state and
```

Reply:
Hm, the grammar is weird but the opening line of [the wikipedia entry](https://en.wikipedia.org/wiki/Protocol_Buffers) seems to think "Protocol Buffers" is a singular proper noun equivalent to "Protobuf". But then uses "are" later on. :S

Reply:
GitHub agrees with you: https://github.com/protocolbuffers/protobuf#overview